PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java

```diff
@@ -534,6 +534,27 @@ private void initializeTableModel(final InteractiveTableModel tableModel) {
                 tableModel.addRow(SparkSubmissionParameter.parameterList[i], configs.containsKey(SparkSubmissionParameter.parameterList[i]) ?
                         configs.get(SparkSubmissionParameter.parameterList[i]) : "");
             }
+
+            for (Map.Entry<String, Object> jobConfigEntry : configs.entrySet()) {
+                String jobConfigKey = jobConfigEntry.getKey();
+                Object jobConfigValue = jobConfigEntry.getValue();
+
+                if (!StringHelper.isNullOrWhiteSpace(jobConfigKey) && !SparkSubmissionParameter.isSubmissionParameter(jobConfigKey)) {
+                    if (jobConfigKey == SparkSubmissionParameter.Conf) {
+                        Map<String, Object> sparkConfigMap;
```

Comment:
Updated. Will send out a new PR

---
In reply to: [117957965](https://github.com/Microsoft/azure-tools-for-java/pull/542#discussion_r117957965) [](ancestors = 117957965)